core-bundle/src/Resources/contao/templates/_new/frontend_module/unfiltered_html.html.twig

```diff
@@ -0,0 +1 @@
+{% extends "@Contao/content_element/unfiltered_html.html.twig" %}
```

Comment:
It's probably a good thing to share templates, because then you only need to adjust a single one to affect both the element and the module. IMHO we could even use the element template in the module controller…

That said - and independent from this PR - I dislike that we have to duplicate things in order to also work as modules. Can we maybe get a general "Content Element Module" to work (using the dynamic parent property of `tl_content`)? :thinking: